### PR TITLE
Merged ingo qwb

### PR DESCRIPTION
**Pull request by Bronson-Brown-deVost**
In this PR I have merged a branch of Ingo's updates to the queries and repo system (largely affecting the text system).
There are many changes in qwb-to-sqe and data-to-sqe; these two packages are used by Ingo to import/synchronize data between SQE and QWB and to import data from other project members, you shouldn't worry about changes in these folders.
There are a few updates to the tests.

### CODE REVIEW

File: Utilities/GenerateTypescriptDTOs/GenerateTypescriptDTOs.csproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.35" />
```
Comment (Bronson-Brown-deVost) [inline Utilities/GenerateTypescriptDTOs/GenerateTypescriptDTOs.csproj]: I am unsure why Dapper is here, the package shouldn't need it.
I will look into removing it.

File: Utilities/GenerateTypescriptInterfaces/GenerateTypescriptInterfaces.csproj [hunk 2]
```diff
@@ -8,6 +8,7 @@
 	<ItemGroup>
 	  <PackageReference Include="CaseExtensions" Version="1.0.3" />
 	  <PackageReference Include="CommandLineParser" Version="2.7.82" />
+	  <PackageReference Include="Dapper" Version="2.0.35" />
```
Comment (Bronson-Brown-deVost) [inline Utilities/GenerateTypescriptInterfaces/GenerateTypescriptInterfaces.csproj]: I am unsure why Dapper is here, the package shouldn't need it.
I will look into removing it.

**Comment by Bronson-Brown-deVost** [comment]
I turns out Dapper was imported into several packages that did not use it.  I have removed these.

File: ConsoleApp1/ConsoleApp1.csproj [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (zmbq) [inline ConsoleApp1/ConsoleApp1.csproj:1]: ConsoleApp1 should probably be removed.

File: data-to-sqe/Program.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (zmbq) [inline data-to-sqe/Program.cs:1]: This also seems redundant, but I am ignoring all data-to-sqe changes....

**Review by zmbq — APPROVED** [review]
This seems fine. I'll approve it, just remember to remove the ConsoleApp1 project that snuck in there.

**Pull request merged**